app/code/Magento/Catalog/Model/ImageUploader.php

```diff
@@ -98,7 +94,8 @@ public function __construct(
         \Psr\Log\LoggerInterface $logger,
         $baseTmpPath,
         $basePath,
-        $allowedExtensions
+        $allowedExtensions,
+        $allowedMimeTypes = []
```

Comment:
If you have an around plugin and $allowedMimeTypes will not set by the plugin it breaks the application.